File: tests/fixtures/php/t103_pass.php
<?php

class UserTest extends \PHPUnit\Framework\TestCase
{
    public function test_create_user_throws_on_empty_name(): void
    {
        $this->expectException(\InvalidArgumentException::class);
        new User("");
    }

    public function test_create_user(): void
    {
        $user = new User("alice");
        $this->assertEquals("alice", $user->getName());
    }
}

File: tests/fixtures/rust/t103_pass.rs
#[test]
#[should_panic(expected = "empty name")]
fn test_create_user_panics_on_empty() {
    User::new("");
}

#[test]
fn test_create_user() {
    let user = User::new("alice");
    assert_eq!(user.name(), "alice");
}
